pkg/util/typeutil/schema.go

```diff
@@ -1593,21 +1628,13 @@ func CreateSparseFloatRowFromMap(input map[string]interface{}) ([]byte, error) {
 				return nil, err
 			}
 
-			var val float64
-			val, ok := v.(float64)
-			if !ok {
-				num, ok := v.(json.Number)
-				if !ok {
-					return nil, fmt.Errorf("invalid value type in JSON: %s", reflect.TypeOf(v))
-				}
-				val, err = strconv.ParseFloat(num.String(), 32)
-				if err != nil {
-					return nil, err
-				}
+			val, err := getValue(v)
```

Comment:
1. is format2 forgotten? the idx(string) can only be parse to uint 32.
2. what abort format1? the idx( "indices": [?, ?, ?] ), please check the input and the result after decode

```
func CreateSparseFloatRowFromJSON(input []byte) ([]byte, error) {
	var vec map[string]interface{}
	decoder := json.NewDecoder(bytes.NewReader(input))
	decoder.DisallowUnknownFields()
	err := decoder.Decode(&vec)
	if err != nil {
		return nil, err
	}
	return CreateSparseFloatRowFromMap(vec)
}
```

Reply:
as [the doc](https://milvus.io/docs/sparse_vector.md#Prepare-sparse-vector-embeddings) described
> The vector dimensions must be of Python int or numpy.integer type, and the values must be of Python float or numpy.floating type.

Reply:
discussed offline and summarizing here:

the referenced doc is only to describe that we support both native Python int and numpy integer type. the accepted value range is described in https://milvus.io/docs/sparse_vector.md#FAQ. so the above 2 cases are actually invalid as the index is out of range.